test/TestApps/MSIXPackager/Package.appxmanifest

```diff
@@ -8,8 +8,8 @@
 
   <Identity
     Name="PushNotificationsWin32App"
-    Publisher="CN=WNP_Reunion"
-    Version="1.0.0.0" />
+    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
```

Comment:
Why a change here was needed? It affects package signing, right?

Reply:
Hi, as paul mentioned in one of the team meetings we are generating the certificate using tools/DevCheck.cmd file which I think is the right approach going forward for automation testing too. @pmpurifoy has more information on this. Thanks.